api/envoy/config/accesslog/v3/accesslog.proto

```diff
@@ -281,6 +284,13 @@ message GrpcStatusFilter {
   bool exclude = 2;
 }
 
+// Filters based on dynamic metadata 'access_log_policy' key. The filter will not log if the key is set to `false`.
```

Comment:
Please move back to qualifying the namespace with `envoy.`. There is value here, we want to make clear which parts of the keyspace belong to Envoy built-ins vs. external. Thanks.

Reply:
Also, it's not really a policy at this point. I think calling this `envoy.access_log_hint` might make sense (since filters and even Envoy's built-in filter are free to ignore).

Reply:
In filter metadata, the keys are placed under namespaces. In this case the key access_log_policy is placed in the namespace "envoy.common" so it doesn't need to be qualified once again I feel. I state this in Well Known Dynamic Metadata docs, but I can also add this in the rbac docs.

Reply:
It's probably worth being explicit here and other places by providing the fully qualified key or at least referencing which namespace it belongs to.

Reply:
I've added more references to the namespace